### ISSUE TITLE
How to remove RISE

### ISSUE THREAD

**Issue by codeAshu** [damianavila/RISE#178]
I was just trying RISE, I already have reveal.js . Now I think it mess up with it? How can I uninstall everything?

**Comment by benrbray** [comment]
+1 this has been bugging me as well.

**Comment by damianavila** [comment]
>I was just trying RISE, I already have reveal.js . Now I think it mess up with it? How can I uninstall everything?

How did you install it?

**Comment by codeAshu** [comment]
by running setup.py . followed the instruction given at the github page.

Regards,
Ashutosh

**Comment by ruffsl** [comment]
Could you comment the livereveal line from your `~/.jupyter/nbconfig/notebook.json` file?
```json
{
  "load_extensions": {
    "livereveal/main": true
  }
}
```

**Comment by diego898** [comment]
I am trying to completely remove rise from my environment. How can I do that?

**Comment by damianavila** [comment]
@diego898 how did you install it?

**Comment by diego898** [comment]
I honestly don't remember :l

**Comment by damianavila** [comment]
@diego898, try `conda list rise`, also give me your notebook version and the output of `jupyter nbextension list`. Thanks.

**Comment by diego898** [comment]
```
➜  ~ conda list rise
# packages in environment at /Users/diego/anaconda:
#

➜  ~ jupyter nbextension list
Known nbextensions:
  config dir: /Users/diego/.jupyter/nbconfig
    notebook section
      python-markdown/main  enabled
      - Validating: OK
      usability/toc/main  enabled
      - Validating: OK
      code_font_size/code_font_size  enabled
      - Validating: OK
      livereveal/main  enabled
      - Validating: OK
      usability/chrome-clipboard/main  enabled
      - Validating: OK
      usability/codefolding/main  enabled
      - Validating: OK
  config dir: /Users/diego/anaconda/etc/jupyter/nbconfig
    notebook section
      jupyter-js-widgets/extension  enabled
      - Validating: OK
      nbpresent/js/nbpresent.min  enabled
      - Validating: OK
      nb_conda/main  enabled
      - Validating: OK
      nb_anacondacloud/main  enabled
      - Validating: OK
    tree section
      nb_conda/tree  enabled
      - Validating: OK
➜  ~
```

**Comment by damianavila** [comment]
Well, I can not see rise there... where are finding rise-related things?

**Comment by diego898** [comment]
I have the following buttons:

![screen shot 2016-10-11 at 8 29 49 am](https://cloud.githubusercontent.com/assets/742407/19276944/e5de8c46-8f8c-11e6-816a-fc7e75ebc6d2.png)

**Comment by damianavila** [comment]
OK, give me the content of this file:   config dir: /Users/diego/.jupyter/nbconfig
You should see a livereveal reference there... if you remove that, you should not see RISE button anymore.

**Comment by diego898** [comment]
That will stop it from loading, but not uninstall it correct?

```
{
  "load_extensions": {
    "usability/toc/main": true,
    "python-markdown/main": true,
    "code_font_size/code_font_size": true,
    "livereveal/main": true,
    "usability/chrome-clipboard/main": true,
    "usability/codefolding/main": true
  }
}
```

**Comment by damianavila** [comment]
Yep, to remove it just look for the livereveal (RISE) files on the paths you get when you run `jupyter --paths`.

**Comment by damianavila** [comment]
Additionally you probably want to eliminate the python side, most probably located under your anaconda path, more specifically under site-packages from your root or env where rise is installed (on the usual places for python things).

It is a little bit hard to clean a previous installation, this is why I now provide pip and conda packages...

**Comment by damianavila** [comment]
Closed by #339
